app/controllers/admin/creator_settings_controller.rb

```diff
@@ -1,15 +1,17 @@
 module Admin
   class CreatorSettingsController < Admin::ApplicationController
-    ALLOWED_PARAMS = %i[community_name logo_svg primary_brand_color_hex invite_only_mode public checked_code_of_conduct
-                        checked_terms_and_conditions].freeze
+    ALLOWED_PARAMS = %i[community_name primary_brand_color_hex invite_only_mode public checked_code_of_conduct
```

Comment:
I renamed the `logo_svg` param to `logo`. There are also some new ones that come with the file upload, `authenticity_token utf8 commit locale`. I'm not sure what `commit` is. `locale` is relevant I imagine if we want to return a localized message (error or info).

Reply:
@nickytonline if we're not using these parameters then I don't think we need to allow them through.

Reply:
Done